src/app/public/plugin-resources/code-examples/angular-tree/advanced/angular-tree-demo.component.html

```diff
@@ -0,0 +1,190 @@
+<div
+  class="sky-margin-stacked-separate"
+>
+
+  <!-- #region [Inputs are for demonstration purposes only.] -->
```

Comment:
What is meant by "Inputs"? Maybe a different way to explain?

Reply:
I updated the description a bit. Is that better?